### PR TITLE
Modular respects .gitignore & .modularignore

### PR DESCRIPTION
**Pull request by AlbertoBrusa**
Workspace resolver will respect `.gitignore`, overridden by `.modularignore` if provided

### CODE REVIEW

File: .changeset/eight-bats-double.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (sgb-io) [inline .changeset/eight-bats-double.md:1]: The workspace resolver package likely needs a changeset for this

Comment (AlbertoBrusa) [reply .changeset/eight-bats-double.md:1]: good catch

File: packages/workspace-resolver/package.json [hunk 2]
```diff
@@ -4,8 +4,8 @@
   "license": "Apache-2.0",
   "main": "src/index.ts",
   "dependencies": {
+    "@esm2cjs/globby": "13.1.4",
```
Comment (sgb-io) [inline packages/workspace-resolver/package.json:7]: What was the need to switch to the forked version of `globby`?

Comment (AlbertoBrusa) [reply packages/workspace-resolver/package.json:7]: OG globby is now a pure ESM package that we can't import

**Review by sgb-io — APPROVED** [review]

**Pull request merged**